src/main.js

```diff
@@ -417,9 +411,9 @@ export default class FeatureService {
       })
   }
 
-  _requestJson (url) {
+  _requestJson(url, fetchOptions) {
     return new Promise((resolve, reject) => {
-      fetch(url)
+      fetch(url, fetchOptions)
```

Comment:
So my question above was about why don't we do `fetch(url, this._esriServiceOptions.fetchOptions)` here, rather than supplying it as an argument in ~5 spots?